src/styles/_variables.scss

```diff
@@ -1,3 +1,5 @@
+@import url("https://fonts.googleapis.com/css2?family=Inter&family=Roboto&display=swap");
```

Comment:
this is already on typography.scss, so isn't needed here

Reply:
deleted